### PR TITLE
Feature/multiple uploads

### PR DESCRIPTION
**Pull request by aarushXL**
completed all the 3 apis where multiple upload were needed, 
designed schema as per stoplight for prescription, instituiton, medicalCenter
added try catch block for custom validation error

### CODE REVIEW

File: src/controllers/prescriptionController.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  }
+};
+
+const AllPrescriptions = async (req, res) => {
+  try {
+    let prescriptions = await PrescriptionServices.getAllPrescriptions();
+
+    if (prescriptions.length === 0) {
+      return res.status(404).json({ message: "No prescription found" });
+    }
```
Comment (afourteia) [inline src/controllers/prescriptionController.js:43]: Don't return 404 on on empty list.

File: src/controllers/prescriptionController.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    // let signed_url;
+    // if (findPrescription.prescription_image) {
+    //   signed_url = getSignedUrl(findPrescription.prescription_image, expiry_time);
+    // }
+    return res.status(200).json({
+      data: { findPrescription },
+      message: "Prescription found.",
+    });
```
Comment (afourteia) [inline src/controllers/prescriptionController.js:88]: use response module

File: src/controllers/prescriptionController.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      });
+    }
+    if (req.files.length > 0) {
+      query.fileLink = fieldNamesList;
+    }
+    let prescription = await PrescriptionServices.createPrescription(query);
+    res.status(200).json({
+      prescription,
+      message: "Institute created successfully",
+    });
```
Comment (afourteia) [inline src/controllers/prescriptionController.js:29]: response module

File: src/schemas/prescriptionSchema.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                        }
+                    } ,
+                    {
+                        price : {
+                            type : Number
+                        }
+                    }
+                ]
+            }
+            }
```
Comment (afourteia) [inline src/schemas/prescriptionSchema.js:69]: make this a separate schema. We will need to read one prescriptionQuote in the future

File: src/schemas/prescriptionSchema.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                    // type : mongoose.Schema.Types.ObjectId,
+                    // ref : "pharmacy"
+                    type : String
+                }
+            },
+            {
+                prescriptionId : {
+                    type : String,
+                }
+            },
```
Comment (afourteia) [inline src/schemas/prescriptionSchema.js:42]: This is mongodDB link to the prescription

File: src/schemas/prescriptionSchema.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+        default: null 
+    },
+    Quotes : {
+        type : [
+            {
+                pharmacyId : {
+                    // type : mongoose.Schema.Types.ObjectId,
+                    // ref : "pharmacy"
+                    type : String
+                }
```
Comment (afourteia) [inline src/schemas/prescriptionSchema.js:36]: mongodb link to the pharmacy schema

**Comment by aarushXL** [comment]
All the changes requested are done, along with the prescription api /

**Review by afourteia — APPROVED** [review]

**Pull request merged**